package.json

```diff
@@ -17,76 +17,78 @@
-    "base62": "2.0.1",
-    "bson": "4.0.0",
-    "debug": "~4.1.0",
-    "diskusage": "^1.1.1",
+    "base62": "^2.0.2",
+    "bson": "^6.8.0",
+    "debug": "^4.3.7",
+    "diskusage": "^1.2.0",
     "fcntl": "github:scality/node-fcntl#0.2.2",
     "hdclient": "scality/hdclient#1.1.7",
     "httpagent": "scality/httpagent#1.0.6",
-    "https-proxy-agent": "^2.2.0",
-    "ioredis": "^4.28.5",
-    "ipaddr.js": "1.9.1",
-    "joi": "^17.6.0",
+    "https-proxy-agent": "^7.0.5",
+    "ioredis": "^5.4.1",
+    "ipaddr.js": "^2.2.0",
+    "joi": "^17.13.3",
     "level": "~5.0.1",
     "level-sublevel": "~6.6.5",
     "mongodb": "^5.2.0",
-    "node-forge": "^1.3.0",
-    "prom-client": "14.2.0",
+    "node-forge": "^1.3.1",
+    "prom-client": "^15.1.3",
     "simple-glob": "^0.2.0",
-    "socket.io": "~4.6.1",
-    "socket.io-client": "~4.6.1",
-    "sproxydclient": "git+https://github.com/scality/sproxydclient#8.0.10",
-    "utf8": "3.0.0",
-    "uuid": "^3.0.1",
-    "werelogs": "scality/werelogs#8.1.4",
-    "xml2js": "~0.4.23"
+    "socket.io": "^4.8.0",
+    "socket.io-client": "^4.8.0",
+    "sproxydclient": "github:scality/sproxydclient#8.1.0",
+    "utf8": "^3.0.0",
+    "uuid": "^10.0.0",
+    "werelogs": "scality/werelogs#8.2.0",
+    "xml2js": "^0.6.2"
   },
   "optionalDependencies": {
     "ioctl": "^2.0.2"
   },
   "devDependencies": {
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-typescript": "^7.16.7",
-    "@sinonjs/fake-timers": "^6.0.1",
-    "@types/ioredis": "^4.28.10",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.21",
-    "@types/xml2js": "^0.4.11",
-    "eslint": "^8.14.0",
-    "eslint-config-airbnb": "6.2.0",
-    "eslint-config-scality": "scality/Guidelines#ec33dfb",
-    "eslint-plugin-react": "^4.3.0",
-    "jest": "^27.5.1",
+    "@babel/preset-env": "^7.25.8",
+    "@babel/preset-typescript": "^7.25.7",
+    "@eslint/compat": "^1.2.0",
+    "@sinonjs/fake-timers": "^13.0.3",
+    "@types/ioredis": "^5.0.0",
+    "@types/jest": "^29.5.13",
+    "@types/node": "^22.7.6",
+    "@types/xml2js": "^0.4.14",
+    "eslint": "^9.12.0",
+    "eslint-config-scality": "scality/Guidelines#8.3.0",
+    "eslint-plugin-react": "^7.37.1",
+    "globals": "^15.11.0",
+    "jest": "^29.7.0",
     "mongodb-memory-server": "^8.12.2",
-    "nyc": "^15.1.0",
-    "sinon": "^9.0.2",
-    "temp": "0.9.1",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.6.0",
-    "typescript": "^4.6.2"
+    "nyc": "^17.1.0",
+    "sinon": "^19.0.2",
+    "temp": "^0.9.4",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.6.3",
+    "typescript-eslint": "^8.10.0"
   },
   "scripts": {
-    "lint": "eslint $(git ls-files '*.js')",
+    "lint": "eslint $(git ls-files '*ts' ' *.js')",
     "lint_md": "mdlint $(git ls-files '*.md')",
     "lint_yml": "yamllint $(git ls-files '*.yml')",
-    "test": "jest tests/unit",
+    "test": "export NODE_OPTIONS=\"--tls-max-v1.2\" && jest tests/unit --detectOpenHandles",
```

Comment:
We should set this option from github actions files, and eventually mention it in a readme or doc, we shouldn't need to change this command